Dockerfile

```diff
@@ -40,14 +40,20 @@ RUN apt-get update && apt-get install -y \
 	createrepo \
 	curl \
 	dpkg-sig \
+	e2fslibs-dev \
 	gcc-mingw-w64 \
 	git \
 	iptables \
 	libapparmor-dev \
 	libcap-dev \
 	libsqlite3-dev \
+	liblzma-dev \
+	libxml2-dev \
+	libz-dev \
 	mercurial \
 	parallel \
+	parted \
```

Comment:
@LK4D4 yes. On a ploop device, a partition and a filesystem is created (so it also calls mkfs.ext4 and tune4fs). Partition creation is optional though, there's a flag whether to create a partition or not, so I can get rid of parted dependency.